src/main/java/org/sagebionetworks/web/client/widget/profile/ProfileImageViewImpl.java

```diff
@@ -16,12 +21,26 @@
 	Icon defaultIcon;
 	@UiField
 	Image image;
+	@UiField
+	Button removePicture;
 	
+	Callback removePictureCallback;
 	Widget widget;
 	
 	@Inject
 	public ProfileImageViewImpl(Binder binder) {
 		widget = binder.createAndBindUi(this);
+		removePicture.addClickHandler(new ClickHandler() {
+			@Override
+			public void onClick(ClickEvent event) {
+				removePictureCallback.invoke();
+			}
+		});
+	}
+	
+	@Override
+	public void setRemovePictureCallback(Callback callback) {
+		this.removePictureCallback = callback;

```

Comment:
View should not hold on to callback.  View should simply call back to presenter (presenter.onRemotePicture()).  Presenter can invoke callback if not null.  
Add test to presenter, to verify callback is invoked.